homeassistant/components/enocean/device.py

```diff
@@ -0,0 +1,41 @@
+"""Representation of an EnOcean device."""
+from enocean.protocol.packet import Packet
+from enocean.utils import combine_hex
+
+from homeassistant.components.enocean.const import (
+    SIGNAL_RECEIVE_MESSAGE,
+    SIGNAL_SEND_MESSAGE,
+)
+from homeassistant.helpers.entity import Entity
+
+
+class EnOceanDevice(Entity):
```

Comment:
Please name it `SomethingEntity`. Ie end with `Entity`.

Reply:
Ok, renamed to EnOceanEntity